src/jobs/native_runtime.html

```diff
@@ -0,0 +1,104 @@
+---
+title: Dart Native Runtime Engineer
+---
+
+<p>The Flutter/Dart runtime team is looking to improve the performance,
+   size and stability provided by the Flutter/Dart Native runtime.
+   To that end, we’re seeking software engineers to join our team and
+   contribute to this effort.</p>
+
+<p>In the “Dart Native Runtime Engineer” role, you’ll be part
+  of a team that is tasked with the following mandates:</p>
+
+<ul>
+  <li>Provide a Dart Virtual Machine runtime that has 
+      the ability to execute Dart code efficiently.
+  </li>
+  <li>Debug and profile Flutter/Dart applications on complex setups.
+  </li>
+  <li>Collaborate closely with Flutter Engine and Framework teams to improve
+      both development and deployment of Flutter applications. 
+  </li>
+  <li>Triage and address issues to make Dart/Flutter apps stable and delightful.
+  </li>
+</ul>
+
+<h2>Job responsibilities</h2>
+
+<ol type="A">
+  <li>Own or assist with one or more of Dart's 
+      virtual machine subsystems.
+  </li>
+  <li>Work with the Flutter team to ensure Flutter apps are performant 
+      and have minimal memory footprint.
+  </li>
+  <li>Participate in shared team responsibilities like issue triage,
+      code review, design review, and bug fixing.
+  </li>
+</ol>
+
+<h2>Job location</h2>
+
+<ul>
+<li>Waterloo, ON, Canada
+<li>Munich, Germany
+</ul>
+
+<h2>Minimum qualifications</h2>
+
+<p><i>You must meet these minimum qualifications to
+   apply for this job</i></p>
+
+<p>
+<ul>
+  <li>Software development experience in one or more general purpose
+      programming languages.
+  </li>
+  <li>Experience working with two or more from the following:
+      Unix/Linux environments, mobile application developement on Android/iOS,
+      developing large software systems.
+  </li>
+  <li>Working proficiency and communication skills in verbal
+      and written English.
+  </li>
+  <li>Enjoyment of working in a team environment.
+  </li>
+  <li>Self-motivation, and a good work ethic.
+  </li>
+</ul>
+</p>
+
+<h2>Preferred qualifications</h2>
+
+<p><i>Having these qualifications is a plus, but transferable
+  skills/experiences may be equally valuable</i></p>
+
+<p>
+<ul>
+<li>Prior experience with programming language implementations.
+</li>
+<li>Experience and drive to work with complex customer setups in order to
+    debug and isolate performance or functional issues.
+</li>
+<li>Experience with Android or iOS runtime environments.
+</li>
+<li>Experience contributing to an open source project.
+</li>
+<li>An enthusiasm for working across teams and building relationships.
+</li>
+</ul>
+</p>
+
+<h2>To apply</h2>
+
+<p>
+Please send the following to
+<a href="mailto:flutter-jobs@google.com">flutter-jobs@google.com</a>:
```

Comment:
Hmm, some of the other pages say:  Please apply at the [Google Careers site]...

Started a thread.

Reply:
ack.